transformers/storage/val/initializer/initializer.go

```diff
@@ -0,0 +1,32 @@
+// VulcanizeDB
+// Copyright © 2018 Vulcanize
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package initializer
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	mcdStorage "github.com/makerdao/vdb-mcd-transformers/transformers/storage"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/val"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/storage"
+)
+
+var valAddress = constants.GetContractAddress("MCD_VAL")
+var StorageTransformerInitializer storage.TransformerInitializer = storage.Transformer{
+	Address:           common.HexToAddress(constants.GetContractAddress("MCD_VAL")),
+	StorageKeysLookup: storage.NewKeysLookup(val.NewKeysLoader(&mcdStorage.MakerStorageRepository{}, valAddress)),
```

Comment:
Not sure what address to use here. This `MCD_VAL` is just a placeholder.

Reply:
Imo it would make sense to just use the actual addresses of the existing Medianizers for now, although we'll probably need to use a different method when we wire these transformers up to track new contracts on the fly.

i.e. 0x64DE91F5A373Cd4c28de3600cB34C7C6cE410C85 and 0x18B4633D6E39870f398597f3c1bA8c4A41294966